scene/resources/curve.cpp

```diff
@@ -762,7 +762,7 @@ Vector2 Curve2D::interpolate_baked(float p_offset, bool p_cubic) const {
 	int pc = baked_point_cache.size();
 	if (pc == 0) {
 		ERR_EXPLAIN("No points in Curve2D");
-		ERR_FAIL_COND_V(pc == 0, Vector2());
+		ERR_FAIL_COND_V(true, Vector2());
```

Comment:
Use `ERR_FAIL_V(Vector2())` if there's no condition.

Reply:
As far as I remember, the condition is actually sort-of important here, as it gets into the error message.

Reply:
I think it doesn't get in the error message if there's an `ERR_EXPLAIN`.

Reply:
However, we could remove the `if (pc == 0) {}` check and keep the rest as is. I guess this is done this way for performance reason (not allocating a string if not needed), but is it that meaningful as an optimization?

Reply:
Changed to `ERR_FAIL_V`.